src/MCPClient/lib/clientScripts/removeUnneededFiles.py

```diff
@@ -2,21 +2,32 @@
 
 import os
 import sys
+import shutil
 
 # databaseFunctions requires Django to be set up
 import django
 django.setup()
 # archivematicaCommon
 from custom_handlers import get_script_logger
 from databaseFunctions import fileWasRemoved
+import ConfigParser
 
-REMOVEABLE_FILES = ["Thumbs.db", "Icon", u"Icon\u000D"]
+clientConfigFilePath = '/etc/archivematica/MCPClient/clientConfig.conf'
+config = ConfigParser.SafeConfigParser()
+config.read(clientConfigFilePath)
+try:
+    removableFiles = [e.strip() for e in config.get('MCPClient', 'removableFiles').split(',')]
+except:
```

Comment:
That's reasonable! We're incrementally trying to improve the codebase, and this is part of it.  https://github.com/artefactual/archivematica/blob/stable/1.5.x/src/MCPClient/lib/clientScripts/indexAIP.py#L37-L41 has a better model for catching the exception.